config/crd/experimental/gateway.networking.k8s.io_gateways.yaml

```diff
@@ -148,13 +148,11 @@ spec:
                 minLength: 1
                 type: string
               infrastructure:
-                description: |+
+                description: |-
```

Comment:
@robscott @youngnick I think this is the reason codegen is failing? Not sure what this indicates

Reply:
🤔 maybe it's tied to go version? Or maybe k8s deps? I think I've seen similar changes recently in other PRs, but wasn't sure what was leading to it

Reply:
According to https://yaml.org/spec/1.2-old/spec.html#id2794534, we're changing from the `+` chomping operator to the `-` chomping operator, which will change the behavior of the literal string such that the final line break and trailing empty lines will be removed (`-`) instead of kept (`+`).

I would guess that the tooling in a specific Go version has changed this somewhere. I've had problems with codegen like this before where my Go version and modules didn't exactly match the ones in the repo.

Personally, I have multiple Go versions installed using https://go.dev/doc/manage-install, and then use the `direnv` tool to have an `.envrc` file that selects the correct version when in that directory, which has seeming stopped this sort of issue happening as much for me. (Visual Studio Code also needs to be poked with the correct version.)

Reply:
Oh actually it's not this file at all; it's the standard version